### ISSUE TITLE
BigQuery Schema page - explain what AF values mean

### ISSUE THREAD

**Issue by mtlynch** [m-lab/m-lab.github.io#204]
On the [BigQuery schema page](http://www.measurementlab.net/data/bq/schema/), there are several address family fields (abbreviated as `_af`), but only some of them explain how the field values translate to address families.

All `_af` fields should include a snippet like the following:
```
AF_INET (IPv4) = 2
AF_INET6 (IPv6) = 10
```

**Comment by iros** [comment]
There are also a lot of 0 values in `client_af`, are those missing values?

![image](https://cloud.githubusercontent.com/assets/18690/16384792/5906216c-3c58-11e6-805d-e9b462a3ae86.png)

**Comment by iros** [comment]
Additionally local_af seems to use 0/1 values, are these different?

![image](https://cloud.githubusercontent.com/assets/18690/16384836/7dc86be0-3c58-11e6-9d62-048408dbf878.png)

**Comment by georgiamoon** [comment]
@pboothe @critzo @mtlynch any thoughts on @iros's question above?

**Comment by mtlynch** [comment]
Thanks for reporting this @iros. This seems like a bug. For `client_af`, `0` should be `AF_UNSPEC` (from `sys/socket.h`) but that seems like a suspiciously high number of `AF_UNSPEC` tests.

Looking at the code, `local_af` actually is implemented differently, where `0` represents IPv4 and `1` represents IPv6. That's confusing, though, and we should fix that.

**Comment by iros** [comment]
Ah! good to know. I'll watch this issue for a fix.

**Comment by critzo** [comment]
@mtlynch Are you suggesting a fix in documentation only, or something additional that needs to occur on the storing of values for `local_af` ?

**Comment by mtlynch** [comment]
I think both. I've updated the original bug text to specify what the current documentation should be, but long term, we should fix the BigQuery dataset so that we're using consistent numbers across AF fields (Google has a bug internally to track this).

**Comment by critzo** [comment]
@pboothe @yachang I'm reviewing old website issues and Michael indicated there was an internal bug tracking the fix described here regarding how `local_af` is implemented. 

Could you review and confirm if this bug is resolved or update this issue when it has been? 
Once that is completed we can update the corresponding documentation.

**Comment by critzo** [comment]
@pboothe @yachang 
Any update on this issue? Just reviewing tickets prioritizing some updates to the website. When you have more info if you couple please update this ticket. Thanks.

**Comment by pboothe** [comment]
This is fixed in the new pipeline.

**Comment by critzo** [comment]
@pboothe I assume that since the new pipeline is now done, that the technical portion of this issue is resolved in the new, date-partitioned tables. Is that correct? And is this now resolved in all tables for new data? Just wanted to clarify before proceeding with documentation updates.

**Comment by pboothe** [comment]
That is correct. It is resolved in the new tables. It is not resolved in the plx.google.com tables. One we re-parse all our old data with the open-source parser, and move fully to the new tables, this problem will go away. I do not know the timeline on that, but @gfr10598 might.

**Comment by critzo** [comment]
Awesome. Thank you @pboothe . As I start to look at the website changes needed here, I have a few more questions that might be better suited in an email thread. Please look for that, and if you or @gfr10598 have thoughts in response that would be great.
